osc-staging.py

```diff
@@ -172,4 +174,9 @@ def do_staging(self, subcmd, opts, *args):
         elif cmd == 'list':
             ListCommand(api).perform()
         elif cmd == 'adi':
-            AdiCommand(api).perform(args[1:], opts.move)
+            if opts.move:
+                AdiCommand(api).perform(args[2:], opts.move)
+            elif opts.by_develproject:
+                AdiCommand(api).perform(args[2:], by_dp=opts.by_develproject)
+            else:
```

Comment:
if with --move argument then args[1] is "--move", doesn't it? if use args[1:] ie. packages actually it's including "--move"?